### PR TITLE
feat: initial implementation

### PR DESCRIPTION
**Pull request by andrewrynhard**

### CODE REVIEW

File: template/.conform.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+          - perf
+          - refactor
+          - style
+          - test
+        scopes:
+          - '*'
+  - type: license
+    spec:
+      skipPaths:
+        - .git/
```
Comment (smira) [inline template/.conform.yaml:29]: I feel we should also skip `testdata/`

File: template/go.mod [hunk 2]
```diff
@@ -0,0 +1 @@
+module github.com/talos-systems/{{ Name }}
```
Comment (smira) [inline template/go.mod:1]: should it also have `go 1.14` ?

File: template/.conform.yaml (hunk 1 above)
Comment (andrewrynhard) [reply template/.conform.yaml:29]: Good idea. I will patch.

File: template/go.mod (hunk 2 above)
Comment (andrewrynhard) [reply template/go.mod:1]: Good catch.

**Pull request merged**